app/mapView/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { Heading, Text } from "@chakra-ui/react";
```

Comment:
I don't feel _too_ strongly either way, but I'm leaning towards Kebab case and naming the file `/map-view/page.tsx` instead of `/mapView/page.tsx`. And that's what you do for th 404 page, so as long as we're consistent I'm happy 😄

Reply:
Done! :)